README.md

```diff
@@ -14,9 +14,9 @@ We are now collecting dubbo user info in order to help us to improve dubbo bette
 
 * [Side projects](http://github.com/dubbo)
     * [Dubbo Spring Boot](https://github.com/dubbo/dubbo-spring-boot-project) - Spring Boot Project for Dubbo.
-    * [Dubbo ops](https://github.com/dubbo/dubbo-ops) - The reference implementation for dubbo ops.
+    * [Dubbo ops](https://github.com/dubbo/dubbo-ops) - The reference implementation for dubbo ops(dubbo-admin,dubbo-monitor-simple,dubbo-registry-simple,etc.).
 * [Developer Mailing list](https://github.com/alibaba/dubbo/issues/1393) - Any questions or suggestions? Subscribe to (dev@dubbo.incubator.apache.org) to discuss with us.
-* [Gitter channel](https://gitter.im/alibaba/dubbo)
-* [Dubbo user manual](http://dubbo.io/books/dubbo-user-book/)
-* [Dubbo developer guide](http://dubbo.io/books/dubbo-dev-book/)
-* [Dubbo admin manual](http://dubbo.io/books/dubbo-admin-book/)
+* [Gitter channel](https://gitter.im/alibaba/dubbo) - Online chat room with Dubbo developers.
+* [Dubbo user manual](http://dubbo.io/books/dubbo-user-book/) - Dubbo User Manual: Describe how to use Dubbo and all features of Dubbo concretely.
```

Comment:
The last three lines may have double words 
` Dubbo User Manual:`
` Dubbo Developer Guide:`
` Dubbo Admin Manual:`
Is it better to delete them?   @kimmking